github/SourceImport.py

```diff
@@ -0,0 +1,163 @@
+import github.GithubObject
```

Comment:
This file is missing the Python source encoding comment (`# -*- coding: utf-8 -*-`) and copyright header.